lib/http_requests.py

```diff
@@ -157,21 +157,33 @@ def get_advisories(branch_name):
         if not current_advisories:
             current_advisories = {}
 
+        override_applied = False
         while depth < MAX_DEPTH:
             depth += 1
             basis_version = yml_data[current_version].get('assembly', {}).get('basis', {}).get('assembly')
             if not basis_version or basis_version in seen:
                 break
 
+            # Check for 'advisories!' override in basis version
+            basis_override_advisories = yml_data[basis_version].get('assembly', {}).get('group', {}).get('advisories!', None)
+            if basis_override_advisories is not None:
+                if not basis_override_advisories:  # If basis version's override_advisories is empty
+                    current_advisories.clear()
+                    break  # No need to process further as the advisories are cleared
+                current_advisories.update(basis_override_advisories)
+
             basis_advisories, _ = process_version_advisories(basis_version, yml_data, seen)
```

Comment:
This part should not be executed if we are overriding the advisories. So either we keep this before the override or you can skip this part, if we are overriding